sign_node/signer.py

```diff
@@ -222,6 +310,122 @@ def check(pkg_path: str) -> typing.Tuple[SignStatusEnum, str]:
     def timedelta_seconds(start_time: datetime, finish_time: datetime) -> int:
         return int((finish_time - start_time).total_seconds())
 
+    @staticmethod
+    def _write_file_content(path: Path, content, mode='w'):
+        with path.open(mode=mode) as fd:
+            fd.write(content)
+
+    @staticmethod
+    def _extract_key_fingerprint(keyid: str) -> str:
+        fingerprint_cmd = plumbum.local['gpg'][
+            '-k',
+            keyid,
+        ]
+        _, stdout, _ = fingerprint_cmd.run()
+
+        key_fingerprint = stdout.split('\n')[1].strip()
```

Comment:
please left a comment why we take second element from array

Reply:
Done, thank you